### DISCUSSION TITLE
「缺陷报告」同一软件不同窗口之间使用panel难以切换

### DISCUSSION THREAD

**Discussion by widon1104** [linuxdeepin/developer-center#3025] [Features Request & Ideas | 特性请求 & 头脑风暴]
系统：deepinV20.2.4
![截图_选择区域_20211001184924](https://user-images.githubusercontent.com/12093211/135610249-88edb92e-2cc4-4311-81a7-30330fd12933.jpg)
就上面这张图，一个chrome，我开了3个窗口，就这张图我就可以指出很多问题
1）chrome图标是黑色，证明我现在窗口选中的就是chrome，只有3个窗口，我总归已经选择了一个窗口吧，但是现在选择的窗口没有任何标识，我可以给出一个方法，就是在当前窗口前面带一个“*“号，这样就可以看出来现在选中的是哪一个窗口了
2）接下来就是恶心的窗口切换问题了，我现在选中了github这个窗口，左键点击百度这个窗口，**很多时候点击了没有任何反应**！然后就是各种恶心的问题，我都不想说了，然后就会出现一个上下箭头组成的图标，点击了有惊喜，然后又跳到其他窗口了。
我觉得窗口切换应该是很简单的事情，就是点击哪一个窗口切换到哪一个窗口就完了，然后不要出现“上下箭头组成的图标“（这个图标出现以后，我无法在panel上面选择软件点击了），希望测试部多测试！

**Comment by kyww** [comment]
Bug修复前先把窗口特效开启一下暂时过渡吧。开了窗口特效就没这个问题。

**Comment by widon1104** [comment]
> Bug修复前先把窗口特效开启一下暂时过渡吧。开了窗口特效就没这个问题。

1）我现在用uos家庭版，希望uos和deepin都能尽快修复这一个bug
2）我不想开启窗口特效，因为我是集显hd620,这个集显很弱，开启窗口特效会导致明显的卡顿，有时候系统甚至几秒钟没有反应，（我的手速较快）为此我发了2个帖子：
https://bbs.deepin.org/post/229615（[改进建议] 希望优化一下窗口特效 ）
https://bbs.chinauos.com/zh/post/10509 （[问题求助] 今天升级系统（2021-12-23）以后系统非常卡顿 ）

**Comment by widon1104** [comment]
我发现关闭窗口特效以后，很多情况下光标都会变成然一个上下箭头组成的图标，只能左键点击了，体验差，希望尽快修复，不要让光标变成箭头图标

**Comment by BLumia** [comment]
应该算是个需求。总记得有个描述一样内容的 issue....
